File: application/mobile/native/apple/ios/ios_x64_copying_framework/iosApp/ViewController.swift
import UIKit
import KotlinFramework

/**
 *
 */
class ViewController: UIViewController {

	/**
	 *
	 */
	override func viewDidLoad() {
		super.viewDidLoad()
		label.text = ExampleKt.exampleValue
	}

	/**
	 *
	 */
	override func didReceiveMemoryWarning() {
		super.didReceiveMemoryWarning()
	}

	/**
	 *
	 */
	@IBOutlet weak var label: UILabel!
}

File: application/mobile/native/apple/ios/ios_x64_without_framework/WithoutFramework/src/main/kotlin/com/company/team/project/application/mobile/native_/apple/ios/ios_x64/ViewController.kt
package com.company.team.project.application.mobile.native_.apple.ios.ios_x64

import kotlinx.cinterop.*
import platform.Foundation.*
import platform.UIKit.*
import com.company.team.project.application.mobile.native_.apple.ios.ios_x64.exampleValue

/**
 *
 */
@ExportObjCClass
class ViewController : UIViewController {

	/**
	 *
	 */
	@OverrideInit
	constructor() : super(nibName = null, bundle = null)

	/**
	 *
	 */
	@OverrideInit
	constructor(coder: NSCoder) : super(coder)

	/**
	 *
	 */
	@ObjCOutlet
	lateinit var label: UILabel

	/**
	 *
	 */
	override fun viewDidLoad() {
		super.viewDidLoad()
		label.text = exampleValue
	}
}
